config/config.go

```diff
@@ -165,3 +189,36 @@ func (c JobConfig) ScrapeInterval() time.Duration {
 func (c JobConfig) ScrapeTimeout() time.Duration {
 	return stringToDuration(c.GetScrapeTimeout())
 }
+
+// TargetGroups returns the encapsulated target groups with sorted targets.
+func (c JobConfig) TargetGroups() (groups []*TargetGroup) {
+	for _, tg := range c.GetTargetGroup() {
+		g := &TargetGroup{
+			Labels: make(clientmodel.LabelSet),
```

Comment:
I think `clientmodel.LabelSet{}` is the accepted style in Prometheus (i.e. only use `make` when there's no alternative).